main.js

```diff
@@ -563,14 +575,28 @@ app.on('ready', function() {
 
       require('open')('mailto:project@openbazaar.org?subject=OpenBazaar Debug Report&body=' + body);
 
-    }},
-    {type: 'separator'},
+    }}
+  ];
+
+  if(launched_from_installer) {
+    template.push({label: 'View Python Error Log', type: 'normal', click: function() {
+      var logPath = __dirname + path.sep + 'python_error.log';
+      require('open')(logPath);
```

Comment:
open is pretty general keyword naming, but I suppose if we're careful with that one.